### ISSUE TITLE
Add timer on questionnaires? 

### ISSUE THREAD

**Issue by anonymous08** [nextcloud/forms#619]
Is it possible to have a timer on every forms? Thanks

**Comment by skjnldsv** [comment]
Hey!
What would the timer do? Because you can already set an expiration date on a form if that is what you mean :)

**Comment by anonymous08** [comment]
(For example) creating exam forms like 1-10 questions. You have to answer it only in 5 mins. Is it possible? Thanks

**Comment by agentlibre** [comment]
I assume the form would be only made available/accessible on a defined date (e.g. 01.08.2022) and for a defined frame of time (e.g. from 14 to 17pm). The form could only be accessed a limited amount of times (or not) and each question could be subject to a timer (e.g. 10 sec). Once the timer is over, the next question will unfold and the user will not be able to return to previous questions.

@anonymous08 Is this the idea?

**Comment by anonymous08** [comment]
@agentlibre Yes, you are correct. (e.g and also Answer all these questions within 30 minutes only. When it reach the time limit, It will automatically submit whether you are finish or not)
